examples/sd_card/sd_card.ino

```diff
@@ -23,13 +45,13 @@
 /*-----------------------------------------------------------------------------
  *  Constant Definitions
  *-----------------------------------------------------------------------------*/
-#define READ_INTERVAL 10000 // interval, in ms, to wait between readings
+const short READ_INTERVAL = 10000; // interval, in ms, to wait between readings
 
-// These pins are used for measurements
-#define UVOUT A0      // Output from the UV sensor
-#define REF_3V3 A1    // 3.3V power on the Arduino board
-#define SD_CS_PIN 10  // CS pin used for SD card reader. Reader should be wired to DIO
-                      // 10, 11, 12, 13
+// Pins used for measurements
+const short UVOUT = A0;      // Output from the UV sensor
+const short REF_3V3 = A1;    // 3.3V power on the Arduino board
```

Comment:
these values aren't relevant to logging - did they get removed from the main repo?